blankiety_galantow/core/room_settings.py

```diff
@@ -0,0 +1,57 @@
+from blankiety_galantow.core.chat import Chat
+from fastapi.logger import logger
+
+
+class RoomSettings:
+    def __init__(self, chat: Chat):
+        self.chat = chat
+        self.name: str = ""
+        self.open: bool = True
+        self.number_of_seats: int = 6
+        self.selecting_time: int = 60
+        self.game_type: str = "default"
+        self.custom_cards: int = 5
+
+    async def update(self, who: str, data: dict):
+        try:
+            await self.set_name(who, data["roomName"])
```

Comment:
Trzeba dodać to pole na froncie, inaczej będzie rzucany wyjątek przez serwer.

Reply:
Na razie może to zakomentuj i Mateusz to odkomentuje w swoim pull requeście.

Reply:
Kurde, byłem pewny, że zakomentowałem tę linijkę przed pushem xD

Reply:
```suggestion
            # await self.set_name(who, data["roomName"])
```